frontend/src/components/single-article/style.css

```diff
@@ -421,7 +419,7 @@ body{
     height: 16px;
   }
 
-  .article-content .date-and-author {
+  .article-content {
```

Comment:
This may affect the whole article-content styling (already defined earlier on Line 370). So may good idea to delete this since this was originally supposed to format specifically for the date and author section which we have since removed.